EDAM_dev.owl

```diff
@@ -60272,12 +60283,12 @@
     <!-- http://edamontology.org/topic_3892 -->
 
     <owl:Class rdf:about="http://edamontology.org/topic_3892">
-        <rdfs:subClassOf rdf:resource="http://edamontology.org/topic_3307"/>
+        <rdfs:subClassOf rdf:resource="http://edamontology.org/topic_0091"/>
         <created_in>1.22</created_in>
         <oboInOwl:hasDefinition>The study and simulation of molecular conformations using a computational model and computer simulations.</oboInOwl:hasDefinition>
         <oboInOwl:inSubset rdf:resource="http://purl.obolibrary.org/obo/edam#edam"/>
         <oboInOwl:inSubset rdf:resource="http://purl.obolibrary.org/obo/edam#topics"/>
-        <rdfs:comment>This includes methods such as Molecular Dynamics, Coarse-grained dynamics, metadynamics, Quantum Mechanics, QM/MM, Markov State Models, etc.</rdfs:comment>
+        <rdfs:comment>This includes methods such as molecular dynamics (MD), coarse-grained dynamics, metadynamics, quantum mechanics, QM/MM, Markov state models, etc.</rdfs:comment>
```

Comment:
```suggestion
        <rdfs:comment>This includes methods such as molecular dynamics (MD), coarse-grained dynamics, metadynamics, quantum mechanics (QM), quantum mechanics/molecular mechanics (QM/MM), Markov state models, etc.</rdfs:comment>
```

Reply:
@matuskalas Shall I commit the suggestion above? I typically propose suggestions when reviewing and do not commit them myself. If you are happy with the minor changes I  propose above, click the "commit suggestion" thingy.